.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -129,7 +126,7 @@ jobs:
 
           - config: 'bpf-next'
             # renovate: datasource=docker depName=quay.io/lvh-images/kind
-            kernel: 'bpf-next-20240625.013127'
+            kernel: '6.6-20240620.102250'
```

Comment:
I recognize this is making the CI results more stable, but it also gets me wondering what if a newer kernel breaks an older release, do we have any early warning system for that? Or we just rely on the kernel breaking the main branch and then figure out whether we need to apply mitigations back to older releases?

Reply:
One more minor interesting note here - 6.6 is the current LTS, but bpf-next is closer to 6.10. This is subtly changing the target tests. Though that said, over the lifetime of maintaining v1.16 it makes sense to test 6.6 (EOL ETA end of 2026)

Reply:
I'll follow up next week with https://github.com/cilium/cilium/issues/33460